### PR TITLE
Feature/unittest/userrepo

### PR DESCRIPTION
**Pull request by JVerbruggen**
## Description

Added unit tests for UserRepository

## Type of change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## Checklist

-   [x] My code follows the style guidelines of this project
-   [x] I did not update API Controllers, if I did, I added/changed Postman tests
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] New and existing unit tests pass locally with my changes
-   [x] I updated the changelog with an end-user readable description
-   [x] I assigned this pull request to the correct project board to update the sprint board

## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Do something here...
2. Then run this test...
3. After that, you should see the result here

## Link to issue

### CODE REVIEW

**Comment by jcoenen96** [comment]
Can you merge develop into this branch? @JVerbruggen

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,7 +14,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added example unittests for other contributors [#74] (https://github.com/DigitalExcellence/dex-backend/issues/74)
 - Added issue & pull request templates (bug & report) - [#11](https://github.com/DigitalExcellence/dex-backend/issues/11)
 - Changed errors to be compliant with RFC 7807, with guids for easy error searching - [#80](https://github.com/DigitalExcellence/dex-backend/issues/80)
-- Added highlighted filter to search endpoint - [#57](https://github.com/DigitalExcellence/dex-backend/issues/57)
+- Added highlighted filter to search endpoint - [#57](https://github.com/DigitalExcellence/dex-backend/issues/57)
+- Added unittests for UserRepository
```
[the remark this answers is not in the document]
Comment (JVerbruggen) [reply CHANGELOG.md:18]: There is no specific issue for the unittests for UserRepository.

[the remark this answers is not in the document]
Comment (jcoenen96) [reply CHANGELOG.md:18]: Create an issue which is something like "Get unit tests in sync with the codebase"

[the remark this answers is not in the document]
Comment (jcoenen96) [reply CHANGELOG.md:18]: Create an issue which is something like "Get unit tests in sync with the codebase"

[the remark this answers is not in the document]
Comment (jcoenen96) [reply CHANGELOG.md:18]: I just noticed Ramon had added
- Setup basic unit test framework to ensure that the core functionality of the application works [#65] (https://github.com/DigitalExcellence/dex-backend/issues/65)
in his pull request. This one is fine for both of you. Just use this one when merging.

File: 3_Repositories.Tests/UserRepositoryTest.cs [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+            User retrieved = await Repository.GetUserAsync(entity.Id);
+
+            Assert.IsTrue(removed);
+            Assert.IsNull(retrieved);
+        }
+
+        /// <summary>
+        /// User that was not in repo not found
+        /// </summary>
+        /// <param name="entity"></param>
```
Comment (jcoenen96) [inline 3_Repositories.Tests/UserRepositoryTest.cs:66]: empty param tag, param should not be called entity

**Pull request merged**